maxim/logger/together/utils.py

```
@@ -0,0 +1,344 @@
+                if hasattr(completion, 'usage') and completion.usage:
+                    parsed_response["usage"] = {
+                        "prompt_tokens": getattr(completion.usage, 'prompt_tokens', 0),
+                        "completion_tokens": getattr(completion.usage, 'completion_tokens', 0),
+                        "total_tokens": getattr(completion.usage, 'total_tokens', 0),
+                    }
+                    print(f"parsed_response: {parsed_response}")
+
+                return parsed_response
+
+            # Fallback for dict-like responses
+            elif isinstance(completion, dict):
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)